app/changelog/kube-state-metrics.yaml

```diff
@@ -1,3 +1,9 @@
+- version: 1.0.5
```

Comment:
Please consider doing this part as separate PR, so this app release can be used in other releases crafted in parallel to 9.0.1

Reply:
@tfussell do you mind if I extract this in separate PR? we need this change also for aws 9.2.2 release